File: pascal.jl
function fact(k)
    fact = 1
    for i in 1:k
        fact = fact * i
    end
    return fact
end

function binomial(n, k)
    return fact(n) ÷ (fact(k) * fact(n - k))
end

row = 4

for i in 0:row
    for j in 0:i
        print(binomial(i, j), " ")
    end
    println()
end

File: pascal.py
def fact(k):
    if k == 0:
        return 1
    return k * fact(k-1)

def binomial(n, k):
    return fact(n) // (fact(k) * fact(n-k))

row = 5

for i in range(row):
    for j in range(i+1):
        print(binomial(i, j), end=' ')
    print()